common/components/internal-header/_internal-header.scss

```diff
@@ -1,311 +1,182 @@
-  @include govuk-clearfix;
-  position: relative;
-  padding-top: govuk-spacing(2);
-}
+  max-width: 1170px;
+  margin-right: govuk-spacing(3);
+  margin-left: govuk-spacing(3);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (min-width: 40.0625em) {
+    margin-right: govuk-spacing(6);
+    margin-left: govuk-spacing(6);
+  }
+
+  @supports (margin: Max(calc(0px))) {
+    margin-right: Max(
+      govuk-spacing(3),
+      calc(govuk-spacing(3) + env(safe-area-inset-right))
+    );
+    margin-left: Max(
+      govuk-spacing(3),
+      calc(govuk-spacing(3) + env(safe-area-inset-left))
+    );
+
+    @media (min-width: 40.0625em) {
+      margin-right: Max(
+        govuk-spacing(6),
+        calc(govuk-spacing(3) + env(safe-area-inset-right))
+      );
+      margin-left: Max(
+        govuk-spacing(6),
+        calc(govuk-spacing(3) + env(safe-area-inset-left))
+      );
+    }
+  }
 
-.app-header__logotype {
-  margin-right: govuk-spacing(1);
+  @media (min-width: 1230px) {
+    margin-right: auto;
+    margin-left: auto;
+  }
 }
 
 .app-header__logotype-crest {
-  margin-right: 1px;
-  fill: currentColor;
-  vertical-align: middle;
+  margin-right: govuk-spacing(2);
+  position: relative;
+  top: -2px;
+  fill: #fff;
 }
 
-.app-header__logotype-crest-fallback-image {
-  width: 42px;
-  height: 35px;
-  border: 0;
-  vertical-align: middle;
+.app-header__logo {
+  padding-right: govuk-spacing(3);
+  display: flex;
+  align-items: center;
 }
 
-.app-header__product-name {
-  @include govuk-font($size: 24);
-  display: inline-table;
+.app-header__logotype {
+  margin-right: govuk-spacing(2);
+  @include govuk-font($size: 24, $weight: bold);
+  display: flex;
+  align-items: center;
 }
 
 .app-header__link {
-  text-decoration: none;
+  @include govuk-font($size: 19);
 
   &:link,
-  &:visited {
-    color: $govuk-header-link;
+  &:visited,
+  &:active {
+    color: #fff;
+    text-decoration: none;
   }
 
   &:hover {
     text-decoration: underline;
   }
 
-  // When focussed, the text colour needs to be darker to ensure that colour
-  // contrast is still acceptable
   &:focus {
-    @include govuk-focused-text;
-  }
-
-  // alphagov/govuk_template includes a specific a:link:focus selector
-  // designed to make unvisited links a slightly darker blue when focussed, so
-  // we need to override the text colour for that combination of selectors.
-  @include govuk-compatibility(govuk_template) {
-    &:link:focus {
-      @include govuk-text-colour;
-    }
-  }
-}
-
-.app-header__link--homepage {
-  // Font size needs to be set on the link so that the box sizing is correct
-  // in Firefox
-  @include govuk-font($size: false, $weight: bold);
-
-  display: inline;
-  font-size: 30px; // We don't have a mixin that produces 30px font size
-  line-height: 1;
-
-  @include mq ($from: desktop) {
-    display: inline-block;
-  }
-
-  &:link,
-  &:visited {
+    outline: 3px solid transparent;
```

Comment:
All of these definitions are actually being applied by using `@include govuk-focused-text;` so maybe let's revert back to that so that if that changes in the future we'll benefit from it.